examples/analog_hamiltonian_simulation/00_Introduction_of_Analog_Hamiltonian_Simulation_with_Rydberg_Atoms.ipynb

```diff
@@ -7,9 +7,9 @@
    "source": [
```

Comment:
<ol><li>"AHS uses ... and tunes ..."</li><li>"all  qubits." [extra space]</li><li>" is  a sequence " [extra space]</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>